maps/southern_cross/southern_cross_jobs.dm

```diff
@@ -113,10 +118,12 @@ var/const/access_explorer = 43
 	faction = "Station"
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "the chief medical officer"
+	supervisors = "the Chief Medical Officer"
 	selection_color = "#515151"
```

Comment:
Maybe have a different selection colour that's not on part with Civvy since SARs respond to medical?